google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -147,7 +147,7 @@ public Project getProject() {
           callback.errorOccurred(GctBundle.getString("appengine.deployment.error.staging.yaml"));
           return Optional.empty();
         }
-        if (runtimeOptional.filter(runtime -> runtime == FlexibleRuntime.custom).isPresent()
```

Comment:
I believe errorprone should be flagging these. IIRC Google Java style is to use .equals even for string and enums.